src/status_im/contexts/settings/wallet/keypairs_and_accounts/encrypted_qr/view.cljs

```diff
@@ -28,7 +28,7 @@
         cleanup-clock                 (rn/use-callback #(set-code nil))
         on-auth-success               (rn/use-callback (fn [entered-password]
                                                          (rf/dispatch
-                                                          [:wallet/get-key-pair-export-connection
+                                                          [:wallet/get-keypair-export-connection
```

Comment:
A thought I had 💭
Maybe we can do a quick review/refactor over the wallet settings code once we finish merging our PRs.
That way we can check for all these spellings and stuff when the codebase has stabilised.

Reply:
Yes maybe when we want to remove FF for accounts and keypair we can check and resolve them in that PR

Reply:
Yeah that sounds good to me 👍